Update configuration settings for 2025.01 (#2056)
neo4j/docs-operations: modules/ROOT/pages/configuration/configuration-settings.adoc

|Valid values
a|One of [MOST_SELECTIVE_LABEL, OFF].
|Default value

> m|+++MOST_SELECTIVE_LABEL+++

Author: Will it break our test if we add label to the def.value cell?

> m|+++MOST_SELECTIVE_LABEL+++

Reviewer: It shouldn't. AFAIK we agreed with Stefano to escape the labels.

> m|+++MOST_SELECTIVE_LABEL+++

Reviewer: If not, we should fix the test and not adapt the content to the tests.

|Description
a|Path to the logging configuration for debug, query, http and security logs.
|Valid values
a|A path. If relative, it is resolved from server.directories.configuration.
|Default value

> m|+++conf/server-logs.xml+++

Author: ```suggestion
m|+++server-logs.xml+++
```

|===

> [role=label--changed-2025.01]

Reviewer: This one as well.

|Description
a|Path to the logging configuration of user logs.
|Valid values
a|A path. If relative, it is resolved from server.directories.configuration.
|Default value

> m|+++conf/user-logs.xml+++

Author: ```suggestion
m|+++user-logs.xml+++
```

|Valid values
a|One of [NONE, ZIP, GZ].
|Default value

> m|+++ZIP+++

Reviewer: Same here - have a label on the default value instead of the whole config.

|===

[role=label--enterprise-edition label--new-2025.01]
[[config_server.directories.configuration]]
=== `server.directories.configuration`

.server.directories.configuration
[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
|===
|Description
a|Root location of the configuration directory.
|Valid values

> a|A path. If relative, it is resolved from server.directories.configuration

Author: ```suggestion
a|A path. If relative, it is resolved from server.directories.neo4j_home.
```
